.ruff.toml

```diff
@@ -38,6 +63,11 @@ extend-ignore = [
 "docs/*.py" = [
   "INP001",  # Implicit-namespace-package. The examples are not a package.
 ]
+"examples/*.py" = [
+  "T201",  # We need print in our examples
+  "E501",  # Line too long
```

Comment:
We should ignore this forever.

Reply:
FOr everything.

Reply:
interesting, I'm not opposed, there's no good reason for the max line length anymore other than readability

Reply:
In an ideal world, the auto formatter deals with line length by itself. It isn't something we need the linter to worry about.

Reply:
yeah reasonable, you'd have that in the pre-commit?

Reply:
That would be the plan in the future.

Reply:
I am also +1 in not having ruff lint line length, we should leave it to a formatter in the future.